BedrockServer.cpp

```diff
@@ -316,6 +316,14 @@ void BedrockServer::sync()
             }
         }
 
+        if ((getState() == SQLiteNodeState::SEARCHING || getState() == SQLiteNodeState::SYNCHRONIZING) &&
+            (preUpdateState != SQLiteNodeState::SEARCHING && preUpdateState != SQLiteNodeState::SYNCHRONIZING)) {
+            if (_commandPortPublic) {
+                SINFO("Closing command port because " << SQLiteNode::stateName(getState()));
+                _commandPortPublic = nullptr;
+            }
```

Comment:
why not use `blockCommandPort`?

Reply:
So I tried that and was having a pain in the ass of a time getting it to work with the tests. It was also more code as I specifically had to add and remove a string from the block reason list when we did this, and then undid it.